majortom_gateway/gateway_api.py

```diff
@@ -10,6 +10,12 @@
 from base64 import b64encode
 import requests
 import hashlib
+try:
+    # python <= 3.7:
```

Comment:
I might flip the order of these (try for the newer library location first), but otherwise lgtm.